src/test/kotlin/minesweeper/domain/BasicCellTest.kt

```diff
@@ -0,0 +1,21 @@
+package minesweeper.domain
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+
+class BasicCellTest : BehaviorSpec({
+    given("BasicCell 은") {
+        val sut = BasicCell(row = 1, column = 1)
+
+        `when`("생성하면") {
+            then("자신의 위치를 알 수 있다") {
+                val result = sut.location()
+                result shouldBe Location(row = 1, column = 1)
```

Comment:
```suggestion
    given("BasicCell 은") {
        val expectedLocation = Location(row = 1, column = 1)
        val row = 1
        val column = 1
        `when`("생성하면") {
            val sut = BasicCell(row = 1, column = 1)
            then("자신의 위치를 알 수 있다") {
                val result = sut.location()
                result shouldBe expectedLocation
```
given,  when을 좀더 분리하면 
조금더 명확한 테스트를 할수 있을거 같아요!